teacher.rb

```diff
@@ -0,0 +1,15 @@
+require './person'
+
+class Teacher
+  def initialize(id, specialization, name = 'Unknown', age = 0, parent_permission: true)
+    @id = id
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+    @specialization = specialization
+  end
```

Comment:
- [ ] Great work implementing the class Teacher :clap: However since a teacher is also a person, it would be nice that the class `Teacher` inherits from class `Person` as it has been done with the class `Student`. Your code could look like this :point_down: 

```ruby
class Teacher < Person
  def initialize(age, specialization, name = "Unknown", parent_permission = true)
    super(age, name, parent_permission)
    @specialization = specialization
  end

  def can_use_services?
    true
  end
end
```